server/src/main/resources/db/migration/V1_209__add_partners_abbreviations.sql

```diff
@@ -14,10 +14,10 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-update partner set abbreviation = 'IOM' where 'name' = 'IOM';
-update partner set abbreviation = 'HIAS' where 'name' = 'HIAS';
-update partner set abbreviation = 'UNHCR' where 'name' = 'UNHCR';
-update partner set abbreviation = 'RP' where 'name' = 'Refuge Point';
-update partner set abbreviation = 'Dignity' where 'name' = 'Dignity for Children';
-update partner set abbreviation = 'CRS' where 'name' = 'Catholic Relief Services';
-update partner set abbreviation = 'TBB' where 'name' = 'Talent Beyond Boundaries';
+update partner set abbreviation = 'IOM' where name = 'IOM';
+update partner set abbreviation = 'HIAS' where name = 'HIAS';
+update partner set abbreviation = 'UNHCR' where name = 'UNHCR';
+update partner set abbreviation = 'RP' where name = 'Refuge Point';
+update partner set abbreviation = 'Dignity' where name = 'Dignity for Children';
+update partner set abbreviation = 'CRS' where name = 'Catholic Relief Services';
+update partner set abbreviation = 'TBB' where name = 'Talent Beyond Boundaries';
```

Comment:
These updates didn't actually set the corresponding abbreviation for me because Postgres says no row matches. (I'm not sure how/why Postgres accepts this as valid SQL when it thinks the quoted column name doesn't exist or but the fix was easy enough... I'm using Postgres 14, so maybe there's a chance my version of Postgres is less-tolerant than the version that originally ran the migration)

Without this fix, migration `V1_210__set_partners_abbreviations_not_null.sql` fails because it's trying to make the `abbreviation` column NON NULL and the rows corresponding to these statements don't have abbreviations.

Reply:
This looks good. Nasty that this got through. Thanks for the pick up.